ingestion-beam/pom.xml

```diff
@@ -47,6 +47,11 @@
             <artifactId>beam-sdks-java-extensions-json-jackson</artifactId>
             <version>${beam.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.beam</groupId>
+            <artifactId>beam-sdks-java-io-amazon-web-services</artifactId>
+            <version>${beam.version}</version>
+        </dependency>
```

Comment:
We probably don't want to merge in this dependency, but the other code in here is probably safe to add for potential future use.

Reply:
i would be okay with merging this dependency and removing it later when the s3 landfill is deleted.